eng/common/templates/post-build/channels/netcore-3-tools.yml

```
@@ -35,6 +35,18 @@ stages:
           artifactName: 'PDBArtifacts'
         continueOnError: true
 
+      # This is necessary whenever we want to publish/restore to an AzDO private feed
+      # Since sdk-task.ps1 tries to restore packages we need to do this authentication here
+      # otherwise it'll complain about accessing a private feed.
```

[nit] Although I like the mental image of the job complaining, maybe something like

```suggestion
      # otherwise there will be authentication failures when accessing a private feed hosted in a different org.
```

makes it more clear what would happen?